### DISCUSSION TITLE
🚨🚨🚨 IMPORTANT: Support/Help requests

### DISCUSSION THREAD

**Discussion by mr-manuel** [Louisvdw/dbus-serialbattery#856] [General]
# This repository is not maintained anymore

Please go to https://github.com/mr-manuel/venus-os_dbus-serialbattery.

-----

Please do not post support/help request for your system under discussions, if you want help from others. If you want to discuss then you are in the correct place.

99,9 % of you do not post all the data/informations we need when opening a discussion and requesting for help. Even if you open a [support request](https://github.com/mr-manuel/venus-os_dbus-serialbattery/issues?assignees=&labels=support&projects=&template=support_request.yml) with a form where all data is requested 50 % of you do not read all.

This costs a lot of time for something that should be more than clear. Anyway if something should not be clear, please let us know how we can improve that.

Thank you for your understanding.

Manuel

# [Click here to open a support request.](https://github.com/mr-manuel/venus-os_dbus-serialbattery/issues?assignees=&labels=support&projects=&template=support_request.yml)

**Comment by ikorpershoek** [comment]
Hello, I'm replacing my current setup with a Raspberry Pi. The driver is working perfectly with my JK-BMS. The RP will be replaced with a Cerbo-s-GX. Can I install the driver on the Cerbo and then switch the cable or would it be better to switch the cable first and then install it on the Cerbo?  I'm testing the new setup and It could take 2 more days to switch to the new setup.

**Reply by mr-manuel** [reply]
You can install the driver and switch the adapter for and back without problems.

**Comment by ikorpershoek** [comment]
Thanks, very much
You made my day :-)
